homeassistant/components/combined_energy/const.py

```diff
@@ -0,0 +1,337 @@
+            state_class=SensorStateClass.MEASUREMENT,
+            native_unit_of_measurement=UnitOfPower.KILO_WATT,
+            device_class=SensorDeviceClass.POWER,
+            entity_registry_enabled_default=False,
+        ),
+        SensorEntityDescription(
+            key="energy_consumed_battery",
+            name="Energy Export Battery",
+            icon="mdi:transmission-tower",
+            state_class=SensorStateClass.TOTAL,
+            native_unit_of_measurement=UnitOfEnergy.WATT_HOUR,
+            device_class=SensorDeviceClass.ENERGY,
+            entity_registry_enabled_default=False,
+        ),
+        SensorEntityDescription(
+            key="power_consumption_battery",
+            name="Power Export Battery",
+            icon="mdi:home-battery",
+            state_class=SensorStateClass.MEASUREMENT,
+            native_unit_of_measurement=UnitOfPower.KILO_WATT,
+            device_class=SensorDeviceClass.POWER,
+            entity_registry_enabled_default=False,
+        ),
+        SensorEntityDescription(
+            key="power_factor_a",
+            name="Power Factor A",
+            state_class=SensorStateClass.MEASUREMENT,
+            native_unit_of_measurement="%",
+            device_class=SensorDeviceClass.POWER_FACTOR,
+        ),
+        SensorEntityDescription(
+            key="power_factor_b",
+            name="Power Factor B",
+            state_class=SensorStateClass.MEASUREMENT,
+            native_unit_of_measurement="%",
+            device_class=SensorDeviceClass.POWER_FACTOR,
+            entity_registry_enabled_default=False,
+        ),
+        SensorEntityDescription(
+            key="power_factor_c",
+            name="Power Factor C",
+            state_class=SensorStateClass.MEASUREMENT,
+            native_unit_of_measurement="%",
+            device_class=SensorDeviceClass.POWER_FACTOR,
+            entity_registry_enabled_default=False,
+        ),
+        SensorEntityDescription(
+            key="voltage_a",
+            name="Voltage A",
+            state_class=SensorStateClass.MEASUREMENT,
+            native_unit_of_measurement="V",
+            device_class=SensorDeviceClass.VOLTAGE,
+        ),
+        SensorEntityDescription(
+            key="voltage_b",
+            name="Voltage B",
+            state_class=SensorStateClass.MEASUREMENT,
+            native_unit_of_measurement="V",
+            device_class=SensorDeviceClass.VOLTAGE,
+            entity_registry_enabled_default=False,
+        ),
+        SensorEntityDescription(
+            key="voltage_c",
+            name="Voltage C",
+            state_class=SensorStateClass.MEASUREMENT,
+            native_unit_of_measurement="V",
+            device_class=SensorDeviceClass.VOLTAGE,
+            entity_registry_enabled_default=False,
+        ),
+    ],
+    "GENERIC_CONSUMER": SENSOR_DESCRIPTIONS_GENERIC_CONSUMER,
+    "ENERGY_BALANCE": SENSOR_DESCRIPTIONS_GENERIC_CONSUMER,
+}
```

Comment:
I'm not 100% but I don't think descriptions should go in const, I think they should stay in sensor.py

Reply:
Looking at the codebase again, it does happen a few times, it's just not super common, so this is probably okay

Reply:
This pattern was taken from the Solar Edge API. 
[Supported overview sensors](https://github.com/home-assistant/core/blob/d106cb48d2cebfc41883e9faf51b2769b8b5dffb/homeassistant/components/solaredge/const.py#L29)